### DISCUSSION TITLE
Asus ROG Strix G15 2021 (HQ403TS-G513QM) has only keyboard and lightbar backlight

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1772] [General]
Describe the bug
It shows all laptop light control setting inside Extra settings, though the model only support keyboard and lightbar control

Clear scenario to Reproduce
Open Extra settings and check, 

Expected behavior
The model doesn't support Lid control and Logo  control but it still shows inside the extra settings!

App Logs
Not needed I guess?

saw similar post and copied what they typed and modified it to discuss the same problem🙏

I posted the picture showing the problem..it's kinda low quality cause i screenshoted it from the video that i shot to keep track of the settings
Pardon me😅 @seerge

![VideoCapture_20231222-134417](https://github.com/seerge/g-helper/assets/154586512/f1862f89-c85d-428c-904b-73dd70bb4449)
![20220509_193425](https://github.com/seerge/g-helper/assets/154586512/e60517f5-c40d-414f-902b-2e41372aec56)

**Comment by seerge** [comment]
You didn't even bother to upload a log and update app to latest version.

**Reply by RAGE-HERO7** [reply]
Bro i am so sorry if i made you angry🙁
I don't have the laptop with me at the moment
I will update the app tomorrow and will send you the log...Although i thought you only needed the model number....nvm So i have to open GHELPER then select LOG option from extra then save it as text document and send here? @seerge

**Reply by seerge** [reply]
Your device is already marked as one with limited-rgb support in latest release

**Reply by RAGE-HERO7** [reply]
Oh Thank You😲...I will defenetly update the app tomorrow and check it out...your hard work is much Appreciated Sir🫡
